app/apps/client/RealAppsEngineUIHost.js

```diff
@@ -15,10 +15,9 @@ export class RealAppsEngineUIHost extends AppsEngineUIHost {
 	}
 
 	getUserAvatarUrl(username) {
-		const avatarUrl = getUserAvatarURL(username);
-
+		let avatarUrl = getUserAvatarURL(username);
```

Comment:
try to avoid `let` usage, usually let leads to more complex and nonreliable code (we need to check we the variable were overwritten)... even more in this small case...doesn't make sense change to let...

btw I would insert this `replace` on the _baseURL attribution... because usually, we make one set operation, and multiple get operations, so would be more performant if you change that code (:

Reply:
Thank you for your careful review and kindly suggestion! It totally makes sense.